ZocBuild.Database.Application/MainWindow.xaml.cs

```diff
@@ -111,7 +112,7 @@ private void btBuild_Click(object sender, RoutedEventArgs e)
                     var fileSystem = new FileSystem();
                     var db = dbSetting.Create(connection, transaction);
                     var gitProcess = new ExternalProcess(pathToGit.FullName);
-                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, false);
+                    var dvcsScriptRepo = new GitScriptRepository(dbSetting.ScriptsPath, dbSetting.ServerName, dbSetting.DatabaseName, gitProcess, fileSystem, sqlParser, new ConsoleLogger(SeverityLevel.Warning), false);

```

Comment:
Ideally, you would implement a `WpfLogger` object for this sample app.  Nothing at ZD uses the sample app (it is just for illustration in case anyone wants to use this open source library), though.